psalm.xml

```diff
@@ -5,6 +5,9 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="psalm-baseline.xml"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
Hmm, we should find code that isn't tested 🤔

Reply:
FYI, in Stdlib, `findUnusedCode` raises an additional 45 new issues - A lot of false negatives for PHPUnit data providers, but otherwise a good chunk of valid issues.

I can amend #84 to include this with an expanded baseline or separate patch?

Reply:
Ref https://github.com/psalm/psalm-plugin-phpunit/issues/131

Also, don't worry about it for now: just something we should do when moving forward :)